web_ui/frontend/app/director/components/DirectorDropdown.tsx

```diff
@@ -0,0 +1,47 @@
+import {Server, StringTree} from "@/index";
+import {Dropdown, InformationSpan} from "@/components";
+import {Box, Typography} from "@mui/material";
+import DirectoryTree from "@/components/DirectoryTree";
+import React from "react";
+
+interface DirectorDropdownProps {
+    server: Server;
+    transition: boolean;
+}
+
+export const DirectorDropdown = ({server, transition} : DirectorDropdownProps) => {
+    return (
+        <Dropdown transition={transition} flexDirection={"column"}>
+            <InformationSpan name={"Type"} value={server.type} />
+            <InformationSpan name={"Status"} value={server.status} />
+            <InformationSpan name={"URL"} value={server.url} />
+            <Typography variant={"body2"} sx={{fontWeight: 500,  display: "inline", mr: 2, my: .5}}>Namespace Prefixes</Typography>
```

Comment:
We probably also want to include
* Server geolocation (`longitude` and `latitude`)
* Server capability (`Caps` field, similar to the origin data exports)